src/index.ts

```diff
@@ -83,6 +130,11 @@ export class TestxServer {
     }
   }
 
+  /**
+   * Bootstraps the Testx server, generating the GraphQL backend and filling in
```

Comment:
```suggestion
   * Bootstraps the TestxServer, generating the GraphQL backend and filling in
```
worth also to mention that it will create the database